big_tests/tests/mam_send_message_SUITE.erl

```diff
@@ -0,0 +1,142 @@
+         mam_pm_send_message/1]).
+
+-import(mam_helper,
+        [stanza_archive_request/2,
+         wait_archive_respond/1,
+         assert_respond_size/2,
+         respond_messages/1,
+         parse_forwarded_message/1]).
+
+-import(distributed_helper, [mim/0,
+                             require_rpc_nodes/1,
+                             rpc/4]).
+
+-include("mam_helper.hrl").
+-include_lib("escalus/include/escalus.hrl").
+-include_lib("escalus/include/escalus_xmlns.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include_lib("exml/include/exml_stream.hrl").
+
+all() ->
+    [{group, send_message}].
+
+groups() ->
+    [
+     {send_message, [], [mam_pm_send_message,
+                         mam_muc_send_message]}
+    ].
+
+domain() ->
+    ct:get_config({hosts, mim, domain}).
+
+%%%===================================================================
+%%% Overall setup/teardown
+%%%===================================================================
+init_per_suite(Config) ->
+    escalus:init_per_suite(Config).
+
+end_per_suite(Config) ->
+    escalus:end_per_suite(Config).
+
+%%%===================================================================
+%%% Group specific setup/teardown
+%%%===================================================================
+init_per_group(Group, Config) ->
+    case mongoose_helper:is_rdbms_enabled(domain()) of
+        true ->
+            load_custom_module(),
+            Config2 = dynamic_modules:save_modules(domain(), Config),
+            rpc(mim(), gen_mod_deps, start_modules, [domain(), group_to_modules(Group)]),
+            Config2;
+        false ->
+            {skip, require_rdbms}
+    end.
+
+end_per_group(_Groupname, Config) ->
+    case mongoose_helper:is_rdbms_enabled(domain()) of
+        true ->
+            dynamic_modules:restore_modules(domain(), Config);
+        false ->
+            ok
+    end,
+    ok.
+
+group_to_modules(send_message) ->
+    MH = muc_light_helper:muc_host(),
+    [{mod_mam_meta, [{backend, rdbms}, {pm, []}, {muc, [{host, MH}]},
+                     {send_message, mam_send_message_example}]},
+     {mod_muc_light, []},
+     {mam_send_message_example, []}].
+
+load_custom_module() ->
+    mam_send_message_example:module_info(),
+    {Mod, Code, File} = code:get_object_code(mam_send_message_example),
+    rpc(mim(), code, load_binary, [Mod, File, Code]).
+
+%%%===================================================================
+%%% Testcase specific setup/teardown
+%%%===================================================================
+
+init_per_testcase(TestCase, Config) ->
+    escalus:init_per_testcase(TestCase, Config).
+
+end_per_testcase(TestCase, Config) ->
+    escalus:end_per_testcase(TestCase, Config).
+
+%%%===================================================================
+%%% Test Cases
+%%%===================================================================
+
+mam_pm_send_message(Config) ->
+    P = ?config(props, Config),
```

Comment:
It doesn't seem to me like this variable has been loaded into the config, isn't it? I think this was one of the mam helper functions, might as well be used here if it is really needed. Same in the test below.

Reply:
it could be undefined.
which still works.
But I want to keep the test similar to how mam_SUITE handles it (i.e. setting this variable and using stanza_archive_request).

Ok, I will set it to 06.